### PR TITLE
jdmize adds Knightdale, NC

### LINKED ISSUE #39: jdmize follows GitHub Flow
# Knightdale, NC

- [ ] create a **branch**
- [ ] add some **commits**
- [ ] open a *pull request*
- [ ] collaborate
- [ ] `merge` with :+1:

### PR DESCRIPTION
**Pull request by jdmize**
This pull request is to add Knightdale, NC to the list of hometowns.

This PR closes #39.

### CODE REVIEW

File: jdmize-knightdale.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+-Miltons
+-El Senor 
+
+### Attractions/ Places to see
+
+=Carolina Mudcats
+
+### Fun Nearby
+
+-Fish
```
Comment (davegthemighty) [inline jdmize-knightdale.md]: Yes to fish!

Comment (davegthemighty) [inline jdmize-knightdale.md]: @jdmize What kind of fish?

Comment (jdmize) [inline jdmize-knightdale.md]: Lake - Bass, Crappie, bream.  2 hour drive to beach.

**Comment by jdmize** [comment]
Leave a comment

**Pull request merged**